integration/containerd/cri/integration-tests.sh

```diff
@@ -318,13 +318,21 @@ TestContainerMemoryUpdate() {
 	testContainerStop
 }
 
+# k8s may restart docker which will impact on containerd stop
+stop_containerd() {
+	local tmp=$(pgrep kubelet || true)
+	[ -n "$tmp" ] && sudo kubeadm reset -f
```

Comment:
I see no relation whatsoever between this script and k8s setup. Calling non-related stuff from various scripts may result in lower maintainability. Sadly I can't say I know a better way to do that.